template/config/default.js

```diff
@@ -0,0 +1 @@
+module.exports = {}
```

Comment:
Could you add a comment here? Something like:

```
/* Warning: do not use this to set values that differ in each environment, only use this for configuration that is the same across all config environments */
```

Reply:
👍